spec/entities/console_spec.rb

```diff
@@ -0,0 +1,123 @@
+# frozen_string_literal: true
+
+RSpec.describe Console do
+  subject(:console) { described_class.new }
+
+  let(:statistic) { Statistic.new }
```

Comment:
don't use let for all spec list of class if you need it in some `describes`.

Here you must declare `let` for all spec class.

If you need some let for special describe - create it inside it.